t/cli/plugin/test_file_logger_std_output.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
```

Comment:
Could we remove the intermediate directory, so this file can be tested without modifying the ci script?